hrm/department_app/models/department.py

```diff
@@ -0,0 +1,14 @@
+"""Django models"""
+from django.db import models
+from djmoney.models.fields import MoneyField
+
+
+class Department(models.Model):
+    """Department"""
+    department = models.CharField('Department',max_length=50)
```

Comment:
Recommendation: Despite Django adds primary keys automatically, it is good practice to explicitly specify such keys in models.

Reply:
Fixed.

Reply:
I don't see any primary keys here, please add primary key to this model